wurst/objects/items/MixingPot/Cure.wurst

```diff
@@ -0,0 +1,38 @@
+package Cure
+
+import PotionsDefinition
+import ID
+import ChannelAbilityPreset
+import ClosureEvents
+import InstantDummyCaster
+import Orders
+import ObjEditingUtils
+
+constant ABIL  = compiletime(ABIL_ID_GEN.next())
+
+//constant ABIL  = "A05K"
+constant ICON  = ICON_PATH.format("LesserClarityPotion")
+constant MODEL = "Models\\PotionGreenSmall.mdl"
+
+constant CAST_RANGE = 500.
+constant TARGET_ALLOWED = "friend,organic,self"
+constant TT = "This potion is said to be able to cure anything. Made from Athelas Seed."
```

Comment:
Don't mention ingredients. That should follow from the recipe and suggests an inverse relationship.